app/views/errors/unprocessable_entity.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= content_for :title, t('page_titles.unprocessable_entity') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= t('page_titles.unprocessable_entity') %></h1>
+    <p class="govuk-body">Try again later</p>
+    <p class="govuk-body">
+      If you have provided correct information and still continue to see this error contact the Becoming a Teacher team: <%= bat_contact_mail_to %>.
```

Comment:
```suggestion
      If you continue to see this error contact the Becoming a Teacher team: <%= bat_contact_mail_to %>.
```

422s should not normally occur to the end user; if they see it they should immediately contact us I think. (This is the content from the 500 page)